### DISCUSSION TITLE
Feature importance 0 for all variables

### DISCUSSION THREAD

**Discussion by agt64** [autogluon/autogluon#1723] [General]
Hi all, 

I trained a model with 80 features, and tested with feature importance with permutation shuffling with approx 1 million rows. The model was optimized on Brier score, and had acceptable AUROC as well (around 0.84). Even with subsample_size = None and num_shuffle_sets = 10, I still get feature importance where every single variable's feature importance is 0. Why would that be? Could it be because of collinearity? 

If that's truly accurate, does that simply mean all variables are equally important? - though I find that hard to believe.

**Comment by Innixma** [comment]
I've never heard this happen before. It shouldn't return all 0s. Could you provide a reproducible example?
